src/modules/EdgeDetect/Module.js

```diff
@@ -7,6 +7,7 @@ module.exports = function edgeDetect(options, UI) {
   options.blur = options.blur || defaults.blur;
   options.highThresholdRatio = options.highThresholdRatio || defaults.highThresholdRatio;
   options.lowThresholdRatio = options.lowThresholdRatio || defaults.lowThresholdRatio;
+  options.hystereis = options.hysteresis || defaults.hysteresis;
```

Comment:
Oops, i think there's a missing `s`? `hysteresis`?

Reply:
I have made a typo where it is implemented as well so it doesn't make any difference. Two negatives make a +ve lol😂. I make a lot of typos. We can open an fto for this later. A good to welcome new contributors.